src/js/bottom/BottomBar.js

```diff
@@ -81,10 +81,16 @@ class BottomBar extends BaseElement {
         this.on('uiPluginClose', () => this.onPluginClose());
 
         // Clicks inside the control bars shouldn't play/pause the player.
-        this.element.addEventListener('click', (e) => {
-            const clickEvent = e || event;
-            clickEvent.stopPropagation();
-        });
+        this.element.addEventListener('click', this.stopPointerPropagation);
+
+        // Double clicks/taps inside the controls shouldn't trigger fullscreen.
+        this.element.addEventListener('dblclick', this.stopPointerPropagation);
+        this.element.addEventListener('touchstart', this.stopPointerPropagation);
```

Comment:
It doesn't use 'this' in stopPointerPropagation but i agree to keep consistency (Plus it's part of the same class) it should bind to this